sql/src/main/java/io/crate/protocols/postgres/types/NameType.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.protocols.postgres.types;
+
+
+import io.netty.buffer.ByteBuf;
+
+import javax.annotation.Nonnull;
+import java.nio.charset.StandardCharsets;
+
+class NameType extends PGType {
+
+    public static final PGType INSTANCE = new NameType();
+    static final int OID = 19;
+
+    private static final int TYPE_LEN = 64;
+    private static final int TYPE_MOD = -1;
+
+    private NameType() {
+        super(OID, TYPE_LEN, TYPE_MOD, "name");
+    }
+
+    @Override
+    public int typArray() {
+        return -1;
+    }
+
+    @Override
+    public int writeAsBinary(ByteBuf buffer, @Nonnull Object value) {
```

Comment:
I want to double check but I think these methods are not needed since we are not using this type.
I will remove if this is the case and provide default implementations raising an exception.